src/filters/index.js

```diff
@@ -61,7 +61,7 @@ var digitsRE = /(\d{3})(?=\d)/g
 exports.currency = function (value, currency) {
   value = parseFloat(value)
   if (!isFinite(value) || (!value && value !== 0)) return ''
-  currency = currency || '$'
+  currency = currency === false ? '' : currency || '$'
```

Comment:
heh, did I ever speak about speed of you coding? :)
is it fine if I force push into this branch? wanna re-create branch. Or better open another PR?